solr/config/schema.xml

```diff
@@ -374,6 +380,55 @@
       </analyzer>
     </fieldtype>
 
+    <!-- Analyzed Text, English -->
+    <fieldtype name="text_en" class="solr.TextField" positionIncrementGap="10000" autoGeneratePhraseQueries="true">
```

Comment:
We're using a `positionIncrementGap` of 100 in content search, should we be doing the same here?